changelogs/unreleased/gh-5272-bootstrap-strategy-auto.md

```diff
@@ -0,0 +1,8 @@
+## feature/replication
+
```

Comment:
```
    Moreover, the replica will check that its box.cfg.replication table
    contains every registered node in the cluster, thus ensuring that it has
    tried to connect to everyone and chose the best bootstrap leader
    possible.
```
I haven't thought of that before, but this will break any non-fullmesh topology, right?

Reply:
Yep. I believe this is intentional.
For non-fullmesh topologies 'manual' and 'supervised' bootstrap strategies are intended.